doc/ledger3.texi

```diff
@@ -512,7 +512,7 @@ Ledger will generate:
 
 @smallexample @c output:66E3A2C
 10-Dec-01 Checking balance      Assets:Checking          $ 1,000.00   $ 1,000.00
-                                Equit:Opening Balances  $ -1,000.00            0
+                                Equity:Opening Balances  $ -1,000.00            0
```

Comment:
On my system (debian wheezy), on branch master, the two commands does not produce the same output. Any idea ?

~/dev/ledger$ python test/DocTests.py -l ./ledger -f doc/ledger3.texi
...............................................................................................................................................................................

~/dev/ledger$ python test/DocTests.py -vv --ledger ./ledger --file ./doc/ledger3.texi | grep 82150D9
82150D9 : Passed

~/dev/ledger$ ctest --output-on-failure -R DocTests
Test project /home/thierry/dev/ledger
    Start 1: DocTestsTest_ledger
1/3 Test #1: DocTestsTest_ledger ..............   Passed    0.14 sec
    Start 2: DocTestsTest_ledger-mode
2/3 Test #2: DocTestsTest_ledger-mode .........   Passed    0.06 sec
    Start 3: DocTestsTest_ledger3
3/3 Test #3: DocTestsTest_ledger3 .............***Failed    4.94 sec
..................................................................................................................E............................................................

The following examples failed:
  82150D9


67% tests passed, 1 tests failed out of 3

Total Test time (real) =   5.17 sec

The following tests FAILED:
	  3 - DocTestsTest_ledger3 (Failed)
Errors while running CTest